azavea/fb-gender-survey-dashboard
# Update About & FAQ text and final polish for mobile

## Overview
Styles the app to work on smaller screens and updates text according to FB's feedback.

Connects #80 

### Demo

Red banner
<img width="1542" alt="Screen Shot 2021-02-01 at 1 04 50 PM" src="https://user-images.githubusercontent.com/5672295/106451372-e89b4680-649f-11eb-8f1d-f9b9a6aa1985.png">

Maxed width
<img width="1676" alt="Screen Shot 2021-02-01 at 3 13 35 PM" src=
"https://user-images.githubusercontent.com/5672295/106451623-416adf00-64a0-11eb-9c2f-55e2c80b6afa.png">

Mobile
<img width="378" alt="Screen Shot 2021-02-01 at 3 13 59 PM" src="https://user-images.githubusercontent.com/5672295/106451652-4b8cdd80-64a0-11eb-84b9-62c04f06278d.png">


<img width="383" alt="Screen Shot 2021-02-01 at 3 13 49 PM" src="https://user-images.githubusercontent.com/5672295/106451650-4af44700-64a0-11eb-9f02-cc88a9892bc5.png">


<img width="379" alt="Screen Shot 2021-02-01 at 3 14 10 PM" src="https://user-images.githubusercontent.com/5672295/106451653-4b8cdd80-64a0-11eb-96dc-79bbfc083c8b.png">

<img width="362" alt="Screen Shot 2021-02-01 at 3 14 25 PM" src="https://user-images.githubusercontent.com/5672295/106451656-4c257400-64a0-11eb-8268-4abcff2a04b3.png">
<img width="395" alt="Screen Shot 2021-02-01 at 3 14 35 PM" src="https://user-images.githubusercontent.com/5672295/106451658-4cbe0a80-64a0-11eb-85e4-b084b7df22c3.png">
<img width="395" alt="Screen Shot 2021-02-01 at 3 14 42 PM" src="https://user-images.githubusercontent.com/5672295/106451659-4cbe0a80-64a0-11eb-81f2-f7c92d738f4b.png">







### Notes
- I didn't go so far as to make the charts fully responsive due to time constraints - mobile users will be able to download nice-looking bar, stacked, and waffle charts, but they'll need to scroll left-right to see the full bar/stacked charts. Waffle charts are stacked.
- Breadcrumbs are hidden on very small screens as they would take up a lot of vertical reestate.
- @TaiWilkin I was able to do half of **043** and changed the color of the notification, but couldn't find where the time was set.
- Set a max-width for elements that were identified in **042**.
- Updated text according to **044, 049, 051, 052, 053, 054, & 055**.

## Testing Instructions
- `git pull`
- View all pages of app and ensure that it looks ok on smaller screens.
- Updated text according to **044, 049, 051, 052, 053, 054, & 055**. Please check my work 😬  I think I got everything!
